### PR TITLE
Enables log collection from both the provisioner pods

### PR DESCRIPTION
**Pull request by keesturam**
Fixes: #720

### CODE REVIEW

File: tests/helpers.py [hunk 1]
```diff
@@ -779,12 +779,13 @@ def get_start_creation_time(interface, pvc_name):
     format = '%H:%M:%S.%f'
     # Get the correct provisioner pod based on the interface
     if interface == constants.CEPHBLOCKPOOL:
-        pod_name = pod.get_rbd_provisioner_pod().name
+        pod_name = pod.get_rbd_provisioner_pod()
```
Comment (RazTamir) [inline tests/helpers.py]: could you please combine get_rbd_provisioner_pod and get_cephfs_provisioner_pod to 'get_provisioner_pod' based on interface?

Comment (keesturam) [reply tests/helpers.py]: Made this change as requested.

File: ocs_ci/ocs/resources/pod.py [hunk 2]
```diff
@@ -357,40 +357,27 @@ def get_ceph_tools_pod():
     return ceph_pod
 
 
-def get_rbd_provisioner_pod():
+def get_csi_provisioner_pod(interface):
```
Comment (ramkiperiy) [inline ocs_ci/ocs/resources/pod.py]: Looks like this function name should be changed, as commented by Raz earlier.

Comment (keesturam) [reply ocs_ci/ocs/resources/pod.py]: I don't understand. do you mean the function name should instead be `get_provisioner_pod` and not `get_csi_provisioner_pod`?

**Comment by keesturam** [comment]
closing this PR as this change has been added with #1235

**Pull request closed without merging**